src/main/java/org/fundacion/jala/converter/config/SwaggerConfig.java

```diff
@@ -5,6 +5,8 @@
  * ("Confidential Information"). You shall not disclose such Confidential
  * Information and shall use it only in accordance with the terms of the
  * license agreement you entered into with Fundacion Jala
+ *
+ * @author Raymundo Guaraguara Sansusty
  */
 package org.fundacion.jala.converter.config;
```

Comment:
A space is missing between line 10 and 11

Reply:
we agreed there would be no space between copyright and he package.